ui/StatusQ/src/StatusQ/Core/Utils/Utils.qml

```diff
@@ -203,15 +203,6 @@ QtObject {
                     `del {` +
                         `text-decoration: line-through;` +
                     `}` +
-                    `table.blockquote td {` +
-                        `padding-left: 10px;` +
-                        `color: ${Theme.palette.baseColor1};` +
-                    `}` +
-                    `table.blockquote td.quoteline {` +
-                        `background-color: ${Theme.palette.baseColor1};` +
-                        `height: 100%;` +
-                        `padding-left: 0;` +
-                    `}` +
```

Comment:
On a second thought, could you please also simplify the NIM code in message/service.nim:

```nim
      of PARSED_TEXT_TYPE_BLOCKQUOTE:
        var
          blockquote = escape_html(parsedText.literal)
          lines = toSeq(blockquote.split(NEW_LINE))
        for i in 0..(lines.len - 1):
          if i + 1 >= lines.len:
            continue
          if lines[i + 1] != "":
            lines[i] = lines[i] & "<br/>"
        blockquote = lines.join("")
        result = result & fmt(
          "<table class=\"blockquote\">" &
            "<tr>" &
              "<td class=\"quoteline\" valign=\"middle\"></td>" &
              "<td>{blockquote}</td>" &
            "</tr>" &
          "</table>")
```

We don't need this to be a `<table>` anymore

Reply:
@caybro updated, now we only get the plain text starting with `blockquote:` so that we can identify it in QML, which is then removed - if you have any better idea, please let know!